src/devices/RpiCpuTemperature/README.md

```diff
@@ -0,0 +1,11 @@
+# RpiCpuTemperature
```

Comment:
I don't think it's supposed to be a binding, put in samples? Waiting for comments from other members😄

Reply:
I think this is fine but it should not be specific to Raspberry Pi - I suspect we will be able to make it work for other boards as well at one point